### ISSUE TITLE
can't render folder title

### ISSUE THREAD

**Issue by StefanoCecere** [jooy2/vitepress-sidebar#82]
i can't render the title of a folder (using its index.md `title` frontmatter )
it always render the folder name. but all childs pages render correctly from fromtmatter or h1.
i tried all combinations. this is my current config (v 1.10.1)
```
        useTitleFromFrontmatter: true,
        useTitleFromFileHeading: true,
        useFolderLinkAsIndexPage: true,
        includeFolderIndexFile: true,
        folderLinkNotIncludesFileName: true,
```

**Comment by jooy2** [comment]
Hello, Thank you for using `vitepress-sidebar`. Also, I apologize for the delay in responding.

The old `useFolderLinkAsIndexPage` option kept the folder name and didn't change it separately.

I just released `1.11.0` with a number of improvements to this option. As mentioned in the issue, we fixed it to show the title of the frontmatter or markdown heading instead of the folder name, and we improved it to check if the index.md file doesn't exist to set the appropriate menu name.

After following the instructions below, can you confirm that the issue is resolved?

1. Upgrade `vitepress-sidebar` to `1.11.0`.
2. `useFolderLinkAsIndexPage` option now deprecated. Rename the `useFolderLinkAsIndexPage` option to `useIndexFileForFolderMenuInfo`.

I'd love to hear back from you to see if the issue has been resolved, and if you like the project, please show your support by leaving a repository star.

Regards,

**Comment by jooy2** [comment]
There has been no response for an extended period of time, so i'm closing this issue.

Please open a new issue if you encounter any new problems. Thank you!
